platform/include/platform/mbed_retarget.h

```diff
@@ -196,268 +219,399 @@ typedef struct DIR_impl DIR;
-#undef  EBADSLT
+#endif
+#ifndef  EBADSLT
 #define EBADSLT         57      /* Invalid slot */
-#undef  EDEADLOCK
+#endif
+#ifndef  EDEADLOCK
 #define EDEADLOCK       EDEADLK /* Resource deadlock would occur */
-#undef  EBFONT
+#endif
+#ifndef  EBFONT
 #define EBFONT          59      /* Bad font file format */
-#undef  ENOSTR
+#endif
+#ifndef  ENOSTR
 #define ENOSTR          60      /* Device not a stream */
-#undef  ENODATA
+#endif
+#ifndef  ENODATA
 #define ENODATA         61      /* No data available */
-#undef  ETIME
+#endif
+#ifndef  ETIME
 #define ETIME           62      /* Timer expired */
-#undef  ENOSR
+#endif
+#ifndef  ENOSR
 #define ENOSR           63      /* Out of streams resources */
-#undef  ENONET
+#endif
+#ifndef  ENONET
 #define ENONET          64      /* Machine is not on the network */
-#undef  ENOPKG
+#endif
+#ifndef  ENOPKG
 #define ENOPKG          65      /* Package not installed */
-#undef  EREMOTE
+#endif
+#ifndef  EREMOTE
 #define EREMOTE         66      /* Object is remote */
-#undef  ENOLINK
+#endif
+#ifndef  ENOLINK
 #define ENOLINK         67      /* Link has been severed */
-#undef  EADV
+#endif
+#ifndef  EADV
 #define EADV            68      /* Advertise error */
-#undef  ESRMNT
+#endif
+#ifndef  ESRMNT
 #define ESRMNT          69      /* Srmount error */
-#undef  ECOMM
+#endif
+#ifndef  ECOMM
 #define ECOMM           70      /* Communication error on send */
-#undef  EPROTO
+#endif
+#ifndef  EPROTO
 #define EPROTO          71      /* Protocol error */
-#undef  EMULTIHOP
+#endif
+#ifndef  EMULTIHOP
 #define EMULTIHOP       72      /* Multihop attempted */
-#undef  EDOTDOT
+#endif
+#ifndef  EDOTDOT
 #define EDOTDOT         73      /* RFS specific error */
-#undef  EBADMSG
+#endif
+#ifndef  EBADMSG
 #define EBADMSG         74      /* Not a data message */
-#undef  EOVERFLOW
+#endif
+#ifndef  EOVERFLOW
 #define EOVERFLOW       75      /* Value too large for defined data type */
-#undef  ENOTUNIQ
+#endif
+#ifndef  ENOTUNIQ
 #define ENOTUNIQ        76      /* Name not unique on network */
-#undef  EBADFD
-#define EBADFD          77      /* File descriptor in bad state */
-#undef  EREMCHG
+#endif
+#ifndef  EBADFD
+#define EBADFD          277      /* File descriptor in bad state */
+#endif
+#ifndef  EREMCHG
 #define EREMCHG         78      /* Remote address changed */
-#undef  ELIBACC
+#endif
+#ifndef  ELIBACC
 #define ELIBACC         79      /* Can not access a needed shared library */
-#undef  ELIBBAD
+#endif
+#ifndef  ELIBBAD
 #define ELIBBAD         80      /* Accessing a corrupted shared library */
-#undef  ELIBSCN
+#endif
+#ifndef  ELIBSCN
 #define ELIBSCN         81      /* .lib section in a.out corrupted */
-#undef  ELIBMAX
+#endif
+#ifndef  ELIBMAX
 #define ELIBMAX         82      /* Attempting to link in too many shared libraries */
-#undef  ELIBEXEC
+#endif
+#ifndef  ELIBEXEC
 #define ELIBEXEC        83      /* Cannot exec a shared library directly */
-#undef  EILSEQ
+#endif
+#ifndef  EILSEQ
```

Comment:
Got issue with this one only for IAR...
Seems EILSEQ is wrongly defined?

$ grep -r EILSEQ *
arm/inc/c/errno.h:  #define EILSEQ  36